src/routes/api/users/getById.js

```diff
@@ -1,16 +1,57 @@
 const logger = require("../../../logger");
-const { createSuccessResponse } = require("../../../response");
+const hash = require("../../../hash");
+const validator = require("validator");
+const {
+  createSuccessResponse,
+  createErrorResponse,
+} = require("../../../response");
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
 
 /**
  * Get a user by their id
  */
-
-// eslint-disable-next-line no-unused-vars
-module.exports = async (req, res, next) => {
+module.exports = async (req, res) => {
```

Comment:
suggestion: We can use `next` instead of `createErrorResponse` to reduce code duplication and improve readability. You can take a look at POST /assets to see how to do this.